src/mjolnir/pbfgraphparser.cc

```diff
@@ -1419,7 +1419,7 @@ struct graph_callback : public OSMPBF::Callback {
 
       // bail if there is nothing bike related
       Tags::const_iterator found;
-      if (!results.has_value() || (found = results->find("amenity")) == results->end() ||
+      if (!results.is_initialized() || (found = results->find("amenity")) == results->end() ||
```

Comment:
has_value is >= 1.68

Reply:
i think you can just use the boolean cast operator